lib/juttle-subprocess.js

```diff
@@ -84,7 +81,7 @@ process.on('message', function(msg) {
 
         var compile_options = {
             stage: 'eval',
-            fg_processors: [implicit_views(config.implicit_view), optimize, views_sourceinfo],
+            implicit_view: config.implicit_view,
```

Comment:
Right I saw that you were following the precedent set in juttle. I was just making sure that it was supposed to be that way and not below a top level property like adapters and juttle-service are.